W-14933482 AEH update edit portal domain jh (#22)
mulesoft/docs-api-experience-hub: modules/ROOT/pages/common-builder-customizations.adoc

From the Builder, you can use the following portal customizations to match your company's brand.

> Access the Builder from the *Manage yur API portal* page by clicking *Preview an publish your portal*.

Reviewer: ```suggestion
Access the Builder from the *Manage your API portal* page by clicking *Preview and publish your portal*.
```
